mappings/net/minecraft/client/gui/screen/pack/DataPackScreen.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/class_5368 net/minecraft/client/gui/screen/pack/DataPackScreen
+	FIELD field_25446 UNKNOWN_PACK_TEXTURE Lnet/minecraft/class_2960;
+	METHOD <init> (Lnet/minecraft/class_437;Lnet/minecraft/class_5359;Ljava/util/function/BiConsumer;Ljava/io/File;)V
+		ARG 1 parent
+		ARG 3 resourcePackManager
```

Comment:
Looks like this arg is on `BiConsumer` param

Reply:
Revert to deobfuscated as the param is not even used